go/ephemeral/user_ek.go

```diff
@@ -288,7 +298,39 @@ func getLatestUserEKMetadataMaybeStale(ctx context.Context, g *libkb.GlobalConte
 		return nil, err
 	}
 
-	return metadata, nil
+	return statement, nil
+}
+
+func getLatestUserEKMetadataMaybeStale(ctx context.Context, g *libkb.GlobalContext) (metadata *keybase1.UserEkMetadata, err error) {
+	defer g.CTrace(ctx, "getLatestUserEKMetadataMaybeStale", func() error { return err })()
+
+	statement, err := getLatestUserEKStatementMaybeStale(ctx, g)
+	if err != nil || statement == nil {
+		return nil, err
+	}
+
+	return &statement.CurrentUserEk, nil
+}
+
+func getAllActiveUserEKs(ctx context.Context, g *libkb.GlobalContext, now keybase1.Time) (usereks []keybase1.UserEkMetadata, err error) {
```

Comment:
nit but for consistency we should call this `metadata` not `userEKs` since there are no private keys..